Tools/MGCB/BuildContent.cs

```diff
@@ -453,6 +466,16 @@ public void Build(out int successCount, out int errorCount)
                     fileAttr = fileAttr & (~FileAttributes.ReadOnly);
                     File.SetAttributes(dest, fileAttr);
 
+                    var buildTime = DateTime.UtcNow - startTime;
+
+                    if (string.IsNullOrEmpty(c.Link))
+                        Console.WriteLine("{0}", c.SourceFile);
+                    else
+                        Console.WriteLine("{0} => {1}", c.SourceFile, c.Link);
```

Comment:
@cra0zy @nkast @Jjagg

We were never logging copy items which meant they were invisible to the build process before.

I used the same logging as built items... but maybe we should log these differently?
```
Copy C:\Souce\File\Input\File.png
Copy C:\Souce\File\Input\Other.png =>Dest\Folder\Other.png
```
I can even fix the output view to show them differently if this makes sense.

Reply:
Also agreed :) 
The Pipeline Tool needs to support parsing this and the link output, I guess it will cause a crash in this branch (not sure how the parsing is implemented).

Reply:
Hmm, for now I would recommend skipping fixing the Pipeline Tool/MGCB output, and instead we should go over MGCB with proper discussion: https://github.com/MonoGame/MonoGame/issues/6243

Reply:
>  I guess it will cause a crash in this branch

@Jjagg - It doesn't.  It appears like a normal "Building" line.

> for now I would recommend skipping fixing the Pipeline Tool/MGCB output

@cra0zy - Sure...  i'm all up for a bigger discussion on the issue.  

For now it is better that we output something instead of nothing, so i would prefer to leave what i have in this PR until we decide on something better.

Reply:
> For now it is better that we output something instead of nothing, so i would prefer to leave what i have in this PR until we decide on something better.

Fix the Pipeline Tools output parser than at least.

Reply:
> Fix the Pipeline Tools output parser than at least.

Fix it to do what?  It doesn't crash or anything... it seems ok as it is.

Reply:
Thats fine if it works, but I thought it would not resolve relative paths or not show the linked path or something like that.

Reply:
> I thought it would not resolve relative paths or not show the linked path or something like that.

It works.  Just says "Building XXXX => XXXX" which isn't technically correct... but works for now.